packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -43,26 +46,33 @@ export class DiagnosticFixPlugin extends Plugin {
         continue;
       }
 
-      const fix = codefixes.getCodeFixes(diagnostic);
-      if (!fix) {
+      const fixes = codefixes.getCodeFixes(diagnostic, true, false);
```

Comment:
The [countdown latch](https://github.com/rehearsal-js/rehearsal-js/pull/653/files#diff-11e3e65b5b4fefbcc62fbc0826e0036b9a8755b79ad29a33ff8149b126a17152R41) is going to be triggered for all `Diagnostic`s that are of the kind `Suggestion` when you filter out [codefixes](https://github.com/rehearsal-js/rehearsal-js/pull/653/files#diff-e3a9a4e5319af2f6b16a9a5b453cb236d0be0ce253205801aacb8851a6ea9953R110)

Please see this example https://tsplay.dev/Wylpnm

I believe you're going to need to unify getting the diagnostics and the codefixes.

Reply:
^^^^ @egorio in your example. What happens when you also have jsDoc comments?

Reply:
Non-any types will be applied, `any` skipped and the comment added. 
Added test cases to cover `any`